assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -146,22 +151,20 @@
 
   .product-block__product-description {
     margin: $gu-v-spacing / 2 0 $gu-v-spacing;
-    font-size: 16px;
     line-height: 1.25;
     font-family: $gu-headline;
   }
 
   .component-feature-list {
-    font-size: 16px;
     line-height: 1.25;
     font-family: $gu-headline;
     list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#ff4e36"/></svg>');
     margin-left: 26px;
-    display: flex;
-    flex-flow: column wrap;
 
-    @include mq($from: leftCol) {
-      height: 200px;
+    @include mq($from: desktop) {
+      display: grid;
```

Comment:
I second this, we should avoid grid until we are ready to roll it out sitewide. We just removed all the grid CSS on our side of the codebase

Reply:
it's a rather simple column stacking for the bullet points. The previous approach was super feeble (it was a flexbox depending on height) and I think this makes it more robust. Older browsers will make it look different but that's okay! it's progressive enhancement.

Reply:
In that case, could you post a screenshot of how it looks in a browser that doesn't support grid? We can't make things look broken for browsers that we support (not saying that this change would do this but it would be good to see what it looks like)

Reply:
good point! Will update the screenshots after the RR demo. FYI it'll revert to the mobile layout (staking all four bullets in a row)

Reply:
all set!